### DISCUSSION TITLE
error in \setlocalecaption

### DISCUSSION THREAD

**Discussion by cookies-lu** [joaomlourenco/novathesis#475] [Q&A]
Hello,

I am trying to compile for the first time the original file template.tex, using pdflatex and this error is appearing:

! LaTeX Error: Command \setlocalecaption undefined.

**Comment by joaomlourenco** [comment]
Please update your LaTeX distribution (probably MikTeX or TexLive).  Search in google how to do it.  
I believe once you update your latex distribution to the latest version, this error will be gone.
Please let me know.
